framework/src/main/java/org/checkerframework/framework/type/BoundsInitializer.java

```diff
@@ -92,11 +94,16 @@ public static void initializeTypeArgs(AnnotatedDeclaredType declaredType) {
             typeArgs.add(typeArg);
 
             // Add mapping from type parameter to the annotated type argument.
-            typeArgMap.put((TypeVariable) typeElement.getTypeParameters().get(i).asType(), typeArg);
+            TypeVariable key =
```

Comment:
there's no need to create a local variable for the key here as the value stored in key is never used in two separate locations. Just directly pass the casted result of `unannotatedType()` into `put()`. Same for line 106 below.

Reply:
@wmdietl suggested to put it in a local variable. 
Link : https://github.com/eisop/checker-framework/pull/12#discussion_r209467638

Reply:
woops, don't worry about this change